jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildAndCacheApplicationLayersStep.java

```diff
@@ -89,9 +89,7 @@
               return cachedLayer;
             }
 
-            LayerBuilder layerBuilder =
-                new LayerBuilder(
-                    sourceFiles, extractionPath, buildConfiguration.getEnableReproducibleBuilds());
+            LayerBuilder layerBuilder = new LayerBuilder(sourceFiles, extractionPath, true);
```

Comment:
Is there any reason to keep this last parameter/reproducibility logic in LayerBuilder?

Reply:
I kept this here since the LayerBuilder is low-level logic, so the trade-off of removing the explicit "true" here would be a comment added to LayerBuilder saying that it only builds reproducibly.

Reply:
Another option could be to rename `LayerBuilder` to `ReproducibleLayerBuilder`, if you want to be explicit.

Reply:
`ReproducibleLayerBuilder` sounds like a good idea